.github/workflows/pr-create-release-build.yml

```diff
@@ -122,92 +121,92 @@ jobs:
         if: ${{ needs.wait-checks.result == 'success' }}
         run: |
           gh pr comment ${{ needs.create-pr.outputs.pr_number }} --body "/lgtm"

```

Comment:
Do we want to do it automatically without reviewing?

Reply:
is this `create-pr` coming from another workflow?

Reply:
Not exactly. While we could exclude it from automation, the initial plan was to incorporate it automatically.